packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -297,6 +297,16 @@ class DateTimePicker extends React.Component {
       const previousDateTime = this.state.dateTime ? this.state.dateTime.clone() : DateTimeUtils.createSafeDate(formattedDate, this.initialTimeZone);
       updatedDateTime = DateTimeUtils.syncDateTime(previousDateTime, date, this.timeValue, this.props.showSeconds);
 
+      if (previousDateTime.isDST() && previousDateTime.hours() === 3) {
+        if (!updatedDateTime.isDST()) {
+          updatedDateTime.subtract(1, 'hours');
```

Comment:
It appears that one hour is subtracted from the time when it should not if the hour is at `03` and changing the date to date light savings. For example.

1. Default date/time is `03/09/2020 03:57`.
2. Change the date to `03/08/2020` The time correctly remains unchanged at `03:57` because it is not ambiguous.
3. Increment or decrement the date again to `03/07/2020` or `03/09/2020`. Notice that the hour is incorrectly adjusted to `02:57` when it is expected to remain `03:57`.


![Sep-12-2022 12-05-35](https://user-images.githubusercontent.com/22618655/189715235-e96bcc01-f7d1-4df8-ab2b-6cf9ebea15e7.gif)

Reply:
Good catch @benbcai - the component should not auto-adjust for values that did not qualify for missing hour, like being set to any 3am value.

Reply:
Fixed [24a4450](https://github.com/cerner/terra-framework/pull/1567/commits/24a4450b2b1795629270c352acdb666c572a7897)